### ISSUE TITLE
Bootloop clover in boot screen

### ISSUE THREAD

**Issue by BASSA89** [daliansky/XiaoMi-Pro#64]
hello, but to update the EFI folder, what do you have to do? just mount EFI with clover and replace boot and clover? because when I restart the computer it gives me the different boot screen and clover keeps me going to bootloop the boot. thank you

**Comment by stevezhengshiqi** [comment]
Hi, you have to delete the origin `BOOT` and `CLOVER` folder and then copy these two folders from newest release. Replace can cause problems; and also make sure /S/L/E and /L/E have no third party kexts.

**Comment by BASSA89** [comment]
thank you very kind, and then already there are I ask you another question .. the folders like alcplugfix, hidpi, etc. are installed via terminal? thank you

**Comment by stevezhengshiqi** [comment]
Yes, hidpi and ALCPlugFix are installed in terminal. For the installation of other folder, I wrote README in each one.

**Comment by BASSA89** [comment]
Thanks a lot

**Comment by BASSA89** [comment]
I'll ask you the last question .. I have a bluetooth key csr8510 a10 but the Mac does not read it ... do you have any ace up its sleeve?

**Comment by stevezhengshiqi** [comment]
@BASSA89 I don't have a BT dongle of your model. You can google it to find the driver. I suggest to buy a supported model in [Rehabman's support list](https://github.com/RehabMan/OS-X-BrcmPatchRAM).

**Comment by BASSA89** [comment]
Thank you Steve

**Comment by BASSA89** [comment]
hello steve, I saw that with the update to mojave does not work the trackpad any more advice?
